### ISSUE TITLE
Use EM or REM for font sizes instead of PX

### ISSUE THREAD

**Issue by easaw** [thunkable/thunkable-issues#473]
Most mobile phones have different screen resolutions, but the physical difference in screen ratio can make sentences and paragraphs (as well as other items) display in unwanted - and unsurprising ways.

Having fonts show in em or rem, could be very helpful when developing for the plethora of resolutions, physical differences and various pixel densities out there.

For now, Thunk staff member Jane has suggested "when Screen opens, get computer height/width of Screen, from Label set Font Size to [appropriate size]" which is actually a neat idea for new projects, though somewhat inconvenient.

The point is that `em` and `rem` have differences which Thunk should look into very carefully. This article should help. https://webdesign.tutsplus.com/tutorials/comprehensive-guide-when-to-use-em-vs-rem--cms-23984

Basically: 

- Translation of `rem` units to pixel value is determined by the font size of the html element. This font size is influenced by inheritance from the browser font size setting unless explicitly overridden with a unit not subject to inheritance.

- Translation of `em` units to pixel values is determined by the font size of the element they’re used on. This font size is influenced by inheritance from parent elements unless explicitly overridden with a unit not subject to inheritance.

The logical choice seems like `rem` as each screen could have a base px size set (16/18?) and then anything with an rem set would auto-adjust (I hope) within that screen 😊

**Comment by josmas** [comment]
Hi @easaw thanks for the feature request and the links, it's very helpful. The main issue is that both `em` and `rem` are browser based units, but there is no such a thing on native mobile. Android does have `sp`, which is similar in nature, but we need to find an abstraction that also covers iOS and web apps, so this is a bit more involved than it may initially seem. We will consider it in the long term in any case. Thanks!

**Comment by easaw** [comment]
Okay, good to know you're looking into it, thank you 😊👍
